wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -215,15 +217,25 @@ public void onClick(View v) {
                 } else if (v == shortcutsPane.getImportPrivateKey()) {
                     SweepWalletActivity.start(WalletActivity.this, true);
                 }
+                else if (v == shortcutsPane.getExplore()) {
+                    startActivity(new Intent(WalletActivity.this, ExploreActivity.class));
+                }
             }
         });
         showHideSecureAction();
+        refreshIfUserHasBalance();
```

Comment:
This doesn't seem like it will be live as the balance can change from zero to non-zero or from non-zero to zero.

This is only called within onCreate.  If the balance changes, then the activity has to be restarted to reflect the correct buttons on this shortcut bar.

Reply:
Fix sync part